src/client.ts

```diff
@@ -634,6 +634,86 @@ export class StreamChat<
     return await this.patch<APIResponse>(this.baseURL + '/app', options);
   }
 
+  /**
+   * Revokes all tokens on application level issued before given time
+   */
+  async revokeTokens(before?: Date | string | null) {
+    if (before === undefined) {
+      before = new Date().toISOString();
+    }
+
+    if (before instanceof Date) {
+      before = before.toISOString();
+    }
+
+    if (before === '') {
+      throw new Error(
+        "Don't pass blank string for since, use null instead if resetting the token revoke",
+      );
+    }
+
+    return await this.updateAppSettings({
+      revoke_tokens_issued_before: before,
+    });
+  }
+
+  /**
+   * Revokes token for a user issued before given time
+   */
+  async revokeUserToken(userID: string, before?: Date | string | null) {
```

Comment:
It's better its multi version to implement this

Reply:
You want to use only the multi version to give this functionality? Could be done added this since requirement mentioned two methods for it depending upon single/multi user.

Reply:
Provide two for convenience but implement the single via the multi version, no need to duplicate implementation.

Reply:
sure, refactored.